Description: Disable failing test on armhf, fails with signal 7
Author: Ximin Luo <infinity0@debian.org>
Bug: https://github.com/rust-lang/rust/issues/40444
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/test/run-pass/packed-struct-vec.rs
+++ b/src/test/run-pass/packed-struct-vec.rs
@@ -8,6 +8,8 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
+// ignore-arm
+
 use std::mem;
 
 #[repr(packed)]
